indico/modules/rb_new/templates/booking_list.html

```diff
@@ -0,0 +1,109 @@
+                            {%- trans %}Book{% endtrans -%}
+                        </a>
+                    </div>
+                </div>
+            {% endif %}
+            {% if has_unlinked_session_blocks %}
+                <div class="section searchable-field">
+                    <div class="text">
+                        <div class="label">
+                            {%- trans %}Session block{% endtrans -%}
+                        </div>
+                        {{ form_row(form.session_block, skip_label=true, hide_description=true) }}
+                    </div>
+                    <div class="toolbar">
+                        <a href="#" target="_blank" class="i-button highlight js-book-btn disabled">
+                            {%- trans %}Book{% endtrans -%}
+                        </a>
+                    </div>
+                </div>
+            {% endif %}
+        </div>
+        {% if reservations %}
+            <div>
+                <h3>Current bookings</h3>
+                <table class="i-table-widget">
+                    <thead>
+                        <tr>
+                            <th class="room-column">{% trans %}Room{% endtrans %}</th>
+                            <th>{% trans %}Reason{% endtrans %}</th>
+                            <th>{% trans %}For whom{% endtrans %}</th>
+                            <th>{% trans %}Date{% endtrans %}</th>
+                            <th class="time-column">{% trans %}Time{% endtrans %}</th>
+                            <th class="link-column">{% trans %}Linked to{% endtrans %}</th>
+                            <th class="action-column"></th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {% for reservation in reservations %}
+                            <tr>
+                                <td>{{ reservation.room.full_name }}</td>
+                                <td>{{ reservation.booking_reason }}</td>
+                                <td>{{ reservation.booked_for_name }}</td>
+                                <td>
+                                    {{ reservation.start_dt | format_date()}}
+                                    {% if reservation.is_repeating %}
+                                        ({% trans %}recurring{% endtrans %})
+                                    {% endif %}
+                                </td>
+                                <td>{{ reservation.start_dt | format_time() }} - {{ reservation.end_dt | format_time() }}</td>
+                                {% set link = reservation.link %}
+                                <td class="link-data">
+                                <span title="{{ link.object.full_title if link.link_type.name == 'session_block' else link.object.title }}">
+                                    {{ link.link_type.name.replace('_', ' ') | capitalize }}
```

Comment:
Please use a dict with translated names for this...

Reply:
Or better, replace the `LinkType` enum with this:

```python
class LinkType(RichIntEnum):
    __title__ = (None, _('Category'), _('Event'), ...)

    category = 1
    event = 2
    contribution = 3
    subcontribution = 4
    session = 5
    session_block = 6
```

Then you can just use `link.link_type.title`